gapis/api/vulkan/api/properties_features_requirements.api

```diff
@@ -547,6 +583,37 @@ sub void GetPhysicalDeviceProperties2(
             }
             case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_ID_PROPERTIES: {
             }
+            case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FLOAT_CONTROLS_PROPERTIES_KHR :{
+              ext := as!VkPhysicalDeviceFloatControlsPropertiesKHR*(next.Ptr)[0]
+              phyDev.PhysicalDeviceFloatControlsPropertiesKHR = new!PhysicalDeviceFloatControlsPropertiesKHR(
+                DenormBehaviorIndependence: ext.denormBehaviorIndependence,
```

Comment:
What is the value of these member variables? They seem to always have these values? Do we expect to store anything else in them in the future?

Reply:
These are useful because *if* the application asked about them, they will show up in the state block.